### PR TITLE
add test proxy

### PR DESCRIPTION
**Pull request by aryszka**
created an initial proxy testing
some go fmt sneaked in, too

### CODE REVIEW

File: proxy/proxytest/proxytest.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"net/http/httptest"
+	"time"
+)
+
+func New(fr filters.Registry, routes ...*eskip.Route) *httptest.Server {
+	dc := testdataclient.New(routes)
+	rt := routing.New(routing.Options{FilterRegistry: fr, DataClients: []routing.DataClient{dc}})
+	pr := proxy.New(rt, proxy.OptionsNone)
+	tsp := httptest.NewServer(pr)
+	time.Sleep(12 * time.Millisecond) // propagate data client routes
```
Comment (mo-gr) [inline proxy/proxytest/proxytest.go]: Isn't there a nicer way? Some channel to listen to for routes to be propagated maybe? This is always just a workaround. Either you don't wait long enough and introduce race conditions in tests or you waste time :-/

**Comment by mo-gr** [comment]
But in general: :+1:

File: proxy/proxytest/proxytest.go (hunk 1 above)
Comment (aryszka) [inline proxy/proxytest/proxytest.go]: true... we already discussed. so far, i've been hesitating to introduce a synchronization point that is only used in tests. #49

**Comment by aryszka** [comment]
ok, i guess i just merge it for now.

**Pull request merged**